package.json

```diff
@@ -109,6 +110,7 @@
     "standard-version": "^9.0.0",
     "text-diff": "^1.0.1",
     "ts-node": "^9.0.0",
+    "typedoc": "^0.20.32",
```

Comment:
typedoc is different to TSDoc (confusingly!) so I don't think we want this. Why did you add it?